### DISCUSSION TITLE
How to compare image that different format in two folders

### DISCUSSION THREAD

**Discussion by jeltaqq** [WinMerge/winmerge#1759] [Q&A] [answered]
I would like to compare the images and find out how they have changed.
The file names are the same and the file extensions are different.
Lets say, folder1 would be ```.png``` and folder2 would be the same files but in ```.jpg```.
I can use WinMerge to compare the images if I rename them match exactly.

Thank you.

**Comment by sdottaka** [comment, marked as answer]
Unfortunately WinMerge cannot compare files with different extensions in folder comparisons.

**Reply by jeltaqq** [reply]
Thanks for letting me know.
